_registry/ignite.apps.appregistry.json

```diff
@@ -30,5 +30,8 @@
   "supportedPlatforms": [
     "mac",
     "linux"
-  ]
+  ],
+  "icon": "https://URL.to/icon",
+  "cover": "https://URL.to/cover",
+  "sha": "003d64525eafbc7265f11f269f9131ffd8d07d71"
```

Comment:
Is this sha from the binary? Maybe in the future, we can create a script to generate the sha from a new version automatically

Reply:
I think the GitHub release hash makes the most sense as it can have multiple binaries

Reply:
Removed the sha property